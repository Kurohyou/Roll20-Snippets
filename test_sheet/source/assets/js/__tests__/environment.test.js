import { describe, expect, it } from 'vitest';

import { environment } from './../testFramework';

describe('Environment',() => {
  it('Should have attributes, characters, query response sub objects and all attributes from sheet in attributes subobject',()=>{
    console.warn('env.attr',environment);
    expect(environment).toMatchSnapshot();
  });
});