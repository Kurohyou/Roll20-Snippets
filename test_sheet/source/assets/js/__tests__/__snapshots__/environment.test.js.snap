// Vitest Snapshot v1

exports[`Environment > Should have attributes, characters, query response sub objects and all attributes from sheet in attributes subobject 1`] = `
{
  "main_tabs_tab": "nav-tabs-main-tabs--settings",
  "template_start": "&{template:test} {{character_name=@{character_name}}} {{character_id=@{character_id}}}",
}
`;
